examples/host_func_test.go

```diff
@@ -19,22 +22,22 @@ type testKey struct{}
 var hostFuncWasm []byte
 
 func Test_hostFunc(t *testing.T) {
-	allocateBuffer := func(context.Context, uint32) uint32 {
```

Comment:
actually I intended this to show how you can stub a function without wazero types.. replacing with `wasm.Function` simplifies for us, but doesn't show how you can do this..